server/api/apps/shared_models/models/quiz_models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
```

Comment:
Translations are not included in the MVP scope currently, so we don't really need to use this